lib/utilities/symbol.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+module.exports = function symbol(name) {
```

Comment:
The goal is for it to be a private symbol, so I copied what ember does. A full symbol polyfil, I believe is a tad trickier.